### PR TITLE
Fixed Autowiring the module repo

### PR DESCRIPTION
**Pull request by larkincunningham**
Made a slight fix...

### CODE REVIEW

**Comment by dlarkinc** [comment]
What is @Autowired?

**Comment by larkincunningham** [comment]
Could have used @Inject, but like Spring's version!

File: src/main/java/ie/cit/soft8027/liveexercise1/service/LearningServiceImpl.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // MISSING!
 public class LearningServiceImpl // MISSING!
 
-	// MISSING!
```
Comment (dlarkinc) [inline src/main/java/ie/cit/soft8027/liveexercise1/service/LearningServiceImpl.java:15]: > Was this really set to this? Who did that silly [thing????](url)

**Review by dlarkinc — APPROVED** [review]
Ok happy now!

**Pull request merged**